src/main/java/com/paskar/email/application/repositiory/CreateAndSaveEmail.java

```diff
@@ -0,0 +1,12 @@
+package com.paskar.email.application.repositiory;
+
+
+import com.paskar.email.application.console.Email;
+
+import java.io.IOException;
+import java.util.List;
+
+public interface CreateAndSaveEmail {
```

Comment:
Better to rename this to EmailStorage/EmailRepository to make it clearer what is this interface about